keps/sig-node/1967-size-memory-backed-volumes/kep.yaml

```diff
@@ -2,6 +2,7 @@ title: Size memory backed volumes
 kep-number: 1967
 authors:
   - "@derekwaynecarr"
+  - "@kannon92" #for stable promotion
 owning-sig: sig-node
 participating-sigs:
   - sig-storage
```

Comment:
Should we also change status (line 8 9) to 'implemented' ?

Reply:
Correct, it's usually done after it was finished.